tests/test_subagent_docstring.py

```
@@ -0,0 +1,193 @@
+                                     "sub_agent still has @mcp.tool() decorator")
+                break
+        else:
+            self.fail("Could not find async def sub_agent")
+
+    def test_doc_assignment_present(self):
+        """sub_agent.__doc__ = _subagent_docstring_for_cli(...) must be present."""
+        assert "sub_agent.__doc__ = _subagent_docstring_for_cli" in self.src
+
+    def test_mcp_add_tool_call_present(self):
+        """mcp.add_tool(sub_agent) must be present (not counting comments)."""
+        code_lines = [l for l in self.src.splitlines()
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)